pages/vi/vi-faq.md

```diff
@@ -134,6 +134,83 @@
     vagrant up
     ```
 
+#### Q18: What if I accidentally resign my manager account, or delete my ole--vagrant-vi folder, how do I recreate my manager profile?
+
++ You can follow these steps:
+
+	1. check the original folder where ole--vagrant-vi lies, if the folder still exists: run 
+	   
+	   ```
+	   cd ole--vagrant-vi
+	   	vagrant global-status
+		vagrant destroy [imagename]
+		cd ..
+		rm -rf ole--vagrant-vi
+		git clone https://github.com/dogi/ole--vagrant-vi.git
+		cd ole--vagrant-vi
+		vagrant up
+		```
+	if not, just deleted the image and recreate it. 
+	
+		```
+		vagrant global-status
+		vagrant destroy [imagename]
+		git clone https://github.com/dogi/ole--vagrant-vi.git
+		cd ole--vagrant-vi
+		vagrant up
+		```
+	2. You may run into error messege in creating virtual image using ```vagrant up ```:
+	 
+	 ```
+	 A Virtual Box machine with the name 'vi' already exists.
+Please use another name or delete the machine with the existing
+name, and try again.
+	 ```
+	 Try running```VBoxManage list vms && VBoxManage list runningvms```
+	 to check out the running virtual images and their names
+	 then identify your vm's name and ID ["my-vm" {c700b8b6-b766-4638-871b-736b44b7db18}]
+	 
+	 + You may not need to delete it: Copy the ID of the desired VM  into the contents of ole--vagrant-vi/.vagrant/machines/default/virtualbox/id. Save the file then run ```vagrant up```. For more information, check out 
+[Vagrant Issues #6623](https://github.com/hashicorp/vagrant/issues/6623).
```

Comment:
Same newline problem here.

Reply:
I checked the mobile version, not sure what's the newline problem. Could you be more specific?

Reply:
Its rendering fine now. Might have been caused due to mix usage of tabs and space, which you fixed.